crates/aptos-dkg/src/pcs/univariate_kzg.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::algebra::homomorphism;
+use ark_ec::{pairing::Pairing, CurveGroup, VariableBaseMSM};
+
+pub struct Map<'a, E: Pairing> {
+    pub lagr_g1: &'a [E::G1Affine],
+}
+
+impl<'a, E: Pairing> homomorphism::Map for Map<'a, E> {
```

Comment:
Comment! 

Explain that the domain consists of one blinding factor (`E::ScalarField`) stored at $f(\omega^0)$ and a vector of evaluations (`Vec<E::ScalarField>`) stored at $f(\omega^i)$'s, for $i=1,\ldots$.